public/UI.js

```diff
@@ -39,7 +39,64 @@ class UIController {
                     </thead>
                     <tbody id="coding-status-body"></tbody>
                 </table>
-            <div id="last-update">Last updated: </div>
+                <div id="last-update">Last updated: </div>
+                <div class="accordion" id="accordionExample">
+                    <div class="card">
+                        <div class="card-heade" id="headingOne">
+                            <h2 class="mb-0">
+                                <button class="btn btn-brown" type="button" data-toggle="collapse" data-target="#collapseOne" aria-expanded="true" aria-controls="collapseOne">
+                                Message Coding table headers Definitions
+                                </button>
+                            </h2>
+                        </div>
+                        <div id="collapseOne" class="collapse show" aria-labelledby="headingOne" data-parent="#accordionExample">
```

Comment:
It would be better if it scrolls up once someone clicks the button.

Reply:
resolved